lib/FileSystemInfo.js

```diff
@@ -2565,6 +2565,7 @@ class FileSystemInfo {
 				}
 				if (
 					snap.timestampHash !== undefined &&
+					current.timestampHash !== undefined &&
```

Comment:
I think this should never be undefined, since it should be computed when its missing.

I think there is some bug somewhere else..